### ISSUE TITLE
Horizontal drag is difficult when timeline is zoomed in to maximum

### ISSUE THREAD

**Issue by onyxfish** [FlowingMedia/TimeFlow#14]
Not being able to scroll the timeline left and right makes me feel oddly some my mouse is broken.  This seems like a pretty common navigation pattern--hope you can add it in a future version.

Thanks.

**Comment by onyxfish** [comment]
Ack, its there, the actual issue is that when zoomed in to a tight (hourly) view, the grip to scroll the timeline almost disappears--becoming small enough that its nearly unclickable.  Maybe the "grips" could be slimmed down a bit to make way for a larger draggable zone.

**Comment by wattenberg** [comment]
Good point about slimming the grips. Hopefully we can fix this one soon!
